source/manual/get-ssh-access.html.md

```diff
@@ -56,16 +56,25 @@ Create a pull request with these changes. Once it has been [reviewed by a member
 [integration-hiera]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata/integration.yaml
 [merging]: /manual/merge-pr.html
 
-## 3. Access remote environments
+## 3. Set up govukcli
+
+Clone the `govuk-aws` repository and add a symlink to make `govukcli` executable globally:
+
+```sh
+cd ~/govuk
+git clone https://github.com/alphagov/govuk-aws
```

Comment:
```suggestion
git clone git@github.com:alphagov/govuk-aws
```

Can we standardize on cloning repos over SSH to make pushing back to them via SSH key easier (rather than having to use access tokens)?

Reply:
Hmm, standardize was maybe the wrong word, there's only this occurrence of `git clone https://...` in the actual docs.

Reply:
Another good suggestion 👍 will do. Thanks!